generators/entity-client/templates/angular/src/main/webapp/app/entities/update/entity-management-update.component.html.ejs

```diff
@@ -79,6 +80,7 @@ _%>
                                 <fa-icon icon="times"></fa-icon>
                             </button>
                         </div>
+                        <!-- TODO: 15964 Incomprehensible expression -->
                         <input type="file" id="file_<%= fieldName %>" data-cy="<%= fieldName %>" (change)="setFileData($event, '<%= fieldName %>', <% if (fieldTypeBlobContent === 'image') { %>true)" accept="image/*" <%= jhiPrefix %>Translate="entity.action.addimage"<% } else { %>false)" <%= jhiPrefix %>Translate="entity.action.addblob"<% } %>/>
```

Comment:
Not sure what's happening here.
The key is simple, but `jhiTranslate` on an input and seems to be always generated.
Looks a bug to me.